reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -4712,6 +4701,17 @@ public int getPrefetch() {
 	}
 
 	/**
+	 * Give a name to this sequence, which can be retrieved using {@link Scannable#name()}
+	 * as long as this is the first reachable {@link Scannable#parents()}.
+	 *
+	 * @param name a name for the sequence
+	 * @return the same sequence, but bearing a name
+	 */
+	public final Flux<T> name(String name) {
+		return FluxNamed.createOrAppend(this, name);
```

Comment:
Not in this exceptional case I think, we don't create a subscriber operator. What should eventually be done is trying to find a name when we create the assembly snapshot exception however.